ledger/acctupdates.go

```diff
@@ -213,6 +213,16 @@ type deferedCommit struct {
 	lookback basics.Round
 }
 
+// RoundOffsetError is an error for when requested round is behind earliest stored db entry
+type RoundOffsetError struct {
+	Round   basics.Round
+	DbRound basics.Round
```

Comment:
rename Round -> requestedRound and DbRound -> dbRound.
we don't need to export the fields here, only the error struct.